src/components/Slider/Slider.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Container } from 'react-bootstrap'
+import SliderItem from './SliderItem/SliderItem'
+
+function Slider({ title }) {
+    const handleClick = (e) => {
+        e.preventDefault();
+        const slider = document.querySelector('.slider');
+        const scrollWidth = slider.scrollWidth;
+        if (e.target.classList.contains('carousel-control-prev') || e.target.classList.contains('carousel-control-prev-icon')) {
```

Comment:
Is there a better way to handleClick from both controls at once without checking for classes?

Reply:
You can always use custom `data` in html elements.
Especially in testing it's useful because you can add eg. data-test-name and not worry too much about classList being polluted or using id's :)

https://developer.mozilla.org/pl/docs/Learn/HTML/Howto/Use_data_attributes